source/index.html.haml

```diff
@@ -83,12 +83,12 @@ description: The days of chasing multiple Linux distributions are over. Standalo
   .container
     .row.largegap
       .col-sm-5.col-sm-offset-1
-        %h3 
+        %h3
           Flatpak can be used with a total of
           = data.distro.count
           distros
         %p
-          See the 
+          See the
```

Comment:
I have just noticed this. I am not sure whether the space character wasn't intentional here. We'll see after it gets deployed. :)

Reply:
I'm also unsure, I have tooling setup to trim all lines, it might be wrong here

Reply:
I am sadly not able to test it because the new build is still not deployed. Probably the same problem as [last time](https://github.com/flatpak/flatpak.github.io/issues/688).